frontend/src/views/CanteenEditor.vue

```diff
@@ -332,6 +378,33 @@ export default {
         delete e["returnValue"]
       }
     },
+    addManager() {
+      this.$refs.managerForm.validate()
+
+      if (!this.managerFormIsValid) {
+        this.$store.dispatch("notifyRequiredFieldsError")
+        return
+      }
+
+      this.$store
+        .dispatch("addManager", {
+          canteenId: this.canteen.id,
+          email: this.newManagerEmail,
+        })
+        .then(() => {
+          this.$store.dispatch("notify", {
+            title: "Mise à jour prise en compte",
+            message: `${this.newManagerEmail} a bien été ajouté`,
+            status: "success",
+          })
+          // avoid calling the GET API again for speed
+          this.managers.push({ email: this.newManagerEmail })
```

Comment:
Une idée à discuter aussi : avoir les gestionnaires et les invités sous deux propriétés différentes. Par ex, cette view "CanteenEditor" peut avoir une _computed property_ managers qui retourne `this.canteen.managers`, ainsi qu'une propriété dans `data` pour `pendingInvitations`.

Ce qu'on gagnerait serait plus de simplicité (car pas besoin du `accountExists`) ainsi qu'une gestion plus naturel car on pourrait séparer les listes de gestionnaires et invités. On pourrait même à terme (pas pour cette PR) ajouter un bouton "_Renvoyer l'email d'invitation_" pour les invités, etc.

----

Option numéro 2 :

Mettre dans l'objet cantine du store les invitations en cours sous une propriété. Ça voudrait dire qu'on enrichit l'objet frontend de la cantine au retour du call  ``/api/v1/provisionalManagers/${canteen.id}`` (fait aujourd'hui dans le _beforeMount_, mais qui passerait sous le store), L'objet finirait par avoir :

```json
{
  "name": "Canteen qqchose",
  ...
  "managers": [{}, {}, ],
  "pendingInvitations": [
    {
      "email": "a@example.com",
      "creationDate": "..." 
    }, 
    {
      ...
    }
  ]
}
```

L'avantage c'est qu'on évite de faire le call API à chaque fois que la view est rendue - on la ferait seulement si la cantine n'a pas déjà l'info. En l'état actuel, à chaque fois que j'ouvrirai le _CantineEditor_ le call API sera fait (même si les infos ont déjà été obtenus).

----

Option numéro 3 :

Ajouter les _pending invitations_ dès la sérialisation de l'objet `Canteen` dans le backend, un peu comme on fait avec les managers. Avantage : pas besoin de faire le call additionnel lors de l'ouverture de cette view. Inconvénient : une jointure en plus lors du get _user canteens_.